templates/includes/sidebar.html

```diff
@@ -0,0 +1,12 @@
+<ul class="sidebar-nav list-group">
+  {% if user.is_authenticated %}
+    <li class="list-group-item">User: <a href="{{ user.get_absolute_url }}">{{ user.get_username }}</a></li>
```

Comment:
The `get_absolute_url` method is not typically available on the user model unless explicitly defined. Ensure that the `User` model has a `get_absolute_url` method or adjust the link accordingly.